broker/extensions.py

```diff
@@ -4,5 +4,5 @@
 from broker.config import config_from_env
 
 config = config_from_env()
-db = SQLAlchemy()
+db = SQLAlchemy(disable_autonaming=True)
```

Comment:
this is a workaround for a new issue with `flask-sqlalchemy` - https://github.com/sqlalchemy/sqlalchemy/discussions/10461